genindex.md

```diff
@@ -0,0 +1,4 @@
+# Index
+
+```{show-index}
```

Comment:
Is this working for you now in both mystmd latex export and jupyterbook?

Reply:
Nope - only with jupyterbook. Sent email before I saw this.